_includes/wiki/it/wikilicense.html

```diff
@@ -1 +1 @@
-La licenza di questo wiki è soggetta a <a href="https://creativecommons.org/licenses/by-sa/4.0/deed" target="_blank" rel="noreferrer">"Creative Commons Attribution-ShareAlike 4.0 International License"</a>.
+La licenza soggettata a <a href="https://creativecommons.org/licenses/by-sa/4.0/deed" target="_blank" rel="noreferrer">"Creative Commons Attribution-ShareAlike 4.0 International License"</a>.
```

Comment:
We are currently rethinking the footer. I think @gilgongo removed this file?

Reply:
I do not see this mods...if you remove its i remove its from my repo...no probl!

Reply:
i review the "Installation for Windows" file and i see that i make some mistake i need to reload the file and remake the pull request...

Reply:
No problem. I can also remove it. The license is now in the footer of the wiki. I'll add it there